code/game/say.dm

```diff
@@ -23,6 +23,9 @@ GLOBAL_LIST_INIT(freqtospan, list(
 		return
 	if(message == "" || !message)
 		return
+	if(findtext(message, "~"))
+		to_chat(src, "<span class='userdanger'>We do not allow the use of tilde on this server</span>")
```

Comment:
.

Reply:
"Tilde use is disabled by this server's config"
But then there isn't a config